examples/webgl_loader_sea3d_morph.html

```diff
@@ -187,6 +182,9 @@
 
 				requestAnimationFrame( animate );
 
+				// Update SEA3D Animations
+				THREE.SEA3D.AnimationHandler.update( delta );
```

Comment:
I'm using `SEA3D.AnimationHandler` and `SEA3D.ScriptHandler` as animation and script collector. It is optional since you can get the `AnimationMixers` manually but It helps to have scene control since it is automated.